test/e2e/run_networking_test.go

```diff
@@ -543,56 +545,58 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 		}
 	})
 
-	It("podman run network bind to 127.0.0.1", func() {
-		slirp4netnsHelp := SystemExec("slirp4netns", []string{"--help"})
-		Expect(slirp4netnsHelp).Should(ExitCleanly())
-		networkConfiguration := "slirp4netns:outbound_addr=127.0.0.1,allow_host_loopback=true"
-		port := strconv.Itoa(GetPort())
-
-		if strings.Contains(slirp4netnsHelp.OutputToString(), "outbound-addr") {
```

Comment:
This part of the test is gone. Fine with me, but on my f40 laptop `slirp4netns --help` shows the two `--outbound-addr` options as `(experimental)`. To me that means "may be removed without notice", and if so the test will now detect that instead of silently passing. Good. Would it be worth a quick slirp4netns PR or issue removing the "experimental"?

Reply:
I think this is more of a may not work warning. I can create a issue there and ask if the experimental warning can be removed there.

Reply:
I see open issues for outbound-addr there so I guess there is a reason to keep it marked experimental